src/utils/__tests__/paths.spec.ts

```diff
@@ -8,6 +8,10 @@ type expectedData = Record<platform, string | Error>;
 
 describe('paths', () => {
```

Comment:
It would be good to assert that `appHome` is an ancestor of `lima` (because we depend on it in factory reset, and it's non-obvious).

Alternatively, delete both in factory reset (slightly duplicate work, but drops the assumption).

Reply:
I added a test to see if `lima` is in any of the other directories, and also delete it if it isn't.